packages/mask/src/extension/popups/constants.ts

```diff
@@ -1,2 +1,3 @@
 export const passwordRegexp = /^(?=.{8,20}$)(?=.*[A-Z])(?=.*[a-z])(?=.*\d)(?=.*[^\dA-Za-z]).*/
 export const MAX_FILE_SIZE = 10 * 1024 * 1024
+export const MAX_GAS_LIMIT = 15000000
```

Comment:
How does it calculate?

Reply:
It's the upper limit set by the designer.